pages/linux/rdesktop.md

```diff
@@ -0,0 +1,28 @@
+# RDESKTOP
+
+> Remote Desktop Protocal client.
+> It can be used to connect remote windows server or PC using RDP protocal.
```

Comment:
Because the rdesktop can connect both the PC or server as long as the remote access is allow.
I will change to server or PC to clear it.